From 4b390fd0e42542239d22bd5a598f1fe7e6acd72c Mon Sep 17 00:00:00 2001
From: vamsikrishnak <vamsikrishnak@ami.com>
Date: Wed, 17 Apr 2024 12:05:39 +0530
Subject: [PATCH] Fix for Redfish Reference Checker

Description:  Redfish Reference Checker getting failed with
errors.

This commit fixes the issue and can able to complete Redfish
Reference checker successfully.

Tested: Tested by running Redfish Reference Checker tool.

Signed-off-by: vamsikrishnak <vamsikrishnak@ami.com>
---
 static/redfish/v1/schema/AMIChassisFRUCollection_v1.xml | 2 +-
 static/redfish/v1/schema/AMINVMeMIPort_v1.xml           | 2 +-
 static/redfish/v1/schema/AMIPcieSwitchCollection_v1.xml | 2 +-
 static/redfish/v1/schema/CupsPolicyCollection_v1.xml    | 2 +-
 static/redfish/v1/schema/CupsPolicy_v1.xml              | 4 ----
 static/redfish/v1/schema/OemAMIChassis_v1.xml           | 4 ++--
 static/redfish/v1/schema/OemLicense_v1.xml              | 2 +-
 static/redfish/v1/schema/OemServiceRoot_v1.xml          | 2 +-
 8 files changed, 8 insertions(+), 12 deletions(-)

diff --git a/static/redfish/v1/schema/AMIChassisFRUCollection_v1.xml b/static/redfish/v1/schema/AMIChassisFRUCollection_v1.xml
index aa129a47..4e459dbc 100644
--- a/static/redfish/v1/schema/AMIChassisFRUCollection_v1.xml
+++ b/static/redfish/v1/schema/AMIChassisFRUCollection_v1.xml
@@ -23,7 +23,7 @@
   <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/RedfishExtensions_v1.xml">
     <edmx:Include Namespace="RedfishExtensions.v1_0_0" Alias="Redfish"/>
   </edmx:Reference>
-  <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/AMIChassisFRU_v1.xml">
+  <edmx:Reference Uri="/redfish/v1/schema/AMIChassisFRU_v1.xml">
     <edmx:Include Namespace="AMIChassisFRU"/>
 	<edmx:Include Namespace="AMIChassisFRU.v1_0_0"/>
   </edmx:Reference>
diff --git a/static/redfish/v1/schema/AMINVMeMIPort_v1.xml b/static/redfish/v1/schema/AMINVMeMIPort_v1.xml
index 1901a66c..55c004b6 100644
--- a/static/redfish/v1/schema/AMINVMeMIPort_v1.xml
+++ b/static/redfish/v1/schema/AMINVMeMIPort_v1.xml
@@ -7,7 +7,7 @@
         <edmx:Include Namespace="Validation.v1_0_0" Alias="Validation"/>
         <edmx:Include Namespace="RedfishExtensions.v1_0_0" Alias="Redfish"/>
     </edmx:Reference>
-    <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/port_v1.xml">
+    <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/Port_v1.xml">
         <edmx:Include Namespace="Port"/>
         <edmx:Include Namespace="Port.v1_3_0"/>
     </edmx:Reference>
diff --git a/static/redfish/v1/schema/AMIPcieSwitchCollection_v1.xml b/static/redfish/v1/schema/AMIPcieSwitchCollection_v1.xml
index 4cea5689..ceccca5d 100644
--- a/static/redfish/v1/schema/AMIPcieSwitchCollection_v1.xml
+++ b/static/redfish/v1/schema/AMIPcieSwitchCollection_v1.xml
@@ -23,7 +23,7 @@
   <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/RedfishExtensions_v1.xml">
     <edmx:Include Namespace="RedfishExtensions.v1_0_0" Alias="Redfish"/>
   </edmx:Reference>
-  <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/AMIPcieSwitch_v1.xml">
+  <edmx:Reference Uri="/redfish/v1/schema/AMIPcieSwitch_v1.xml">
     <edmx:Include Namespace="AMIPcieSwitch"/>
   </edmx:Reference>
 
diff --git a/static/redfish/v1/schema/CupsPolicyCollection_v1.xml b/static/redfish/v1/schema/CupsPolicyCollection_v1.xml
index d5e17a37..1cb4a145 100644
--- a/static/redfish/v1/schema/CupsPolicyCollection_v1.xml
+++ b/static/redfish/v1/schema/CupsPolicyCollection_v1.xml
@@ -19,7 +19,7 @@
   <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/RedfishExtensions_v1.xml">
     <edmx:Include Namespace="RedfishExtensions.v1_0_0" Alias="Redfish"/>
   </edmx:Reference>
-  <edmx:Reference Uri="/redfish/v1/schema/CupsPolicy.xml">
+  <edmx:Reference Uri="/redfish/v1/schema/CupsPolicy_v1.xml">
     <edmx:Include Namespace="CupsPolicy"/>
     <edmx:Include Namespace="CupsPolicy.v1_0_0"/>
   </edmx:Reference>
diff --git a/static/redfish/v1/schema/CupsPolicy_v1.xml b/static/redfish/v1/schema/CupsPolicy_v1.xml
index be453a59..50a74d08 100644
--- a/static/redfish/v1/schema/CupsPolicy_v1.xml
+++ b/static/redfish/v1/schema/CupsPolicy_v1.xml
@@ -24,10 +24,6 @@
     <edmx:Include Namespace="Resource"/>
     <edmx:Include Namespace="Resource.v1_0_0"/>
   </edmx:Reference>
-  <edmx:Reference Uri="/redfish/v1/schema/CupsSensor_v1.xml">
-    <edmx:Include Namespace="CupsSensor"/>
-    <edmx:Include Namespace="CupsSensor.v1_0_0"/>
-  </edmx:Reference>
   <edmx:Reference Uri="/redfish/v1/schema/CupsPolicy_v1.xml">
     <edmx:Include Namespace="CupsPolicy"/>
     <edmx:Include Namespace="CupsPolicy.v1_0_0"/>
diff --git a/static/redfish/v1/schema/OemAMIChassis_v1.xml b/static/redfish/v1/schema/OemAMIChassis_v1.xml
index d2f823a3..2c25094a 100644
--- a/static/redfish/v1/schema/OemAMIChassis_v1.xml
+++ b/static/redfish/v1/schema/OemAMIChassis_v1.xml
@@ -18,11 +18,11 @@
       <edmx:Include Namespace="Resource"/>
       <edmx:Include Namespace="Resource.v1_0_0"/>
     </edmx:Reference>
-    <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/AMIChassisFRUCollection_v1.xml">
+    <edmx:Reference Uri="/redfish/v1/schema/AMIChassisFRUCollection_v1.xml">
       <edmx:Include Namespace="AMIChassisFRUCollection"/>
       <edmx:Include Namespace="AMIChassisFRUCollection.v1_0_0"/>
     </edmx:Reference>
-    <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/AMIChassisFRU_v1.xml">
+    <edmx:Reference Uri="/redfish/v1/schema/AMIChassisFRU_v1.xml">
       <edmx:Include Namespace="AMIChassisFRU"/>
       <edmx:Include Namespace="AMIChassisFRU.v1_0_0"/>
     </edmx:Reference>
diff --git a/static/redfish/v1/schema/OemLicense_v1.xml b/static/redfish/v1/schema/OemLicense_v1.xml
index 839c65ea..698e9711 100644
--- a/static/redfish/v1/schema/OemLicense_v1.xml
+++ b/static/redfish/v1/schema/OemLicense_v1.xml
@@ -23,7 +23,7 @@
 		<edmx:Include Namespace="Resource"/>
 		<edmx:Include Namespace="Resource.v1_0_0"/>
 	</edmx:Reference>
-	<edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/ProvisionDynamicFeature_v1.xml">
+	<edmx:Reference Uri="/redfish/v1/schema/ProvisionDynamicFeature_v1.xml">
 		<edmx:Include Namespace="ProvisionDynamicFeature"/>
 	</edmx:Reference>
 	<edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/RedfishExtensions_v1.xml">
diff --git a/static/redfish/v1/schema/OemServiceRoot_v1.xml b/static/redfish/v1/schema/OemServiceRoot_v1.xml
index 62a3bf22..39ddfa7e 100644
--- a/static/redfish/v1/schema/OemServiceRoot_v1.xml
+++ b/static/redfish/v1/schema/OemServiceRoot_v1.xml
@@ -19,7 +19,7 @@
     <edmx:Include Namespace="CupsService"/>
   </edmx:Reference>
 
-   <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/AMIPcieSwitchCollection_v1.xml">
+  <edmx:Reference Uri="/redfish/v1/schema/AMIPcieSwitchCollection_v1.xml">
       <edmx:Include Namespace="AMIPcieSwitchCollection"/>
       <edmx:Include Namespace="AMIPcieSwitchCollection.v1_0_0"/>
    </edmx:Reference>
-- 
2.43.2

